src/engine/renderer/tr_shader.cpp

```diff
@@ -5396,7 +5396,16 @@ static shader_t *FinishShader()
 		// keep only the first stage
 		stages[1].active = false;
 		shader.numStages = 1;
-		strcat(shader.name, "$depth");
+
+		const char *shaderSuffix = "$depth";
+		if ( strlen(shader.name) + strlen(shaderSuffix) >= MAX_QPATH )
+		{
+			Log::Warn( "Shader name %s%s exceeds MAX_QPATH", shader.name, shaderSuffix );
```

Comment:
Let's also tell them what MAX_QPATH is.

Reply:
Do you mean to print `MAX_QPATH` value?

Reply:
Yes

Reply:
I did that, I also rewrote the patch to allow shaders to have length of `MAX_QPATH` + "$depth", except it's checked against MAX_QPATH everywhere it's not a depth shader, so we even not invalidate the depth shader if the non-depth shader has valid path name.